clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/loadbalancing/GoogleHttpLoadBalancer.groovy

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.google.model.loadbalancing
+
+import groovy.transform.Canonical
+
+@Canonical
+class GoogleHttpLoadBalancer extends BaseGoogleHttpLoadBalancer {
+  static final googleLoadBalancerType = GoogleLoadBalancerType.HTTP
```

Comment:
`GoogleHttpLoadBalancer.googleLoadBalancerType` is redundant. I suggest dropping the "googleLoadBalancer" to just "type" - it's cleaner ;-)